packages/sku/src/services/vite/helpers/config/createConfig.ts

```diff
@@ -36,8 +37,17 @@ export const createServerBuildConfig = (skuContext: SkuContext) => {
 
 export const createClientBuildConfig = (skuContext: SkuContext) => {
   const outDir = createOutDir(skuContext.paths.target);
+
+  const isProductionBuild = process.env.NODE_ENV === 'production';
+  const isIntegration = process.env.SKU_INTEGRATION === 'true';
+  const bundleAnalyzer =
+    isProductionBuild || !isIntegration
+      ? [bundleAnalyzerPlugin({ name: 'client' })]
+      : [];
+
   return createSkuViteConfig(
     {
+      plugins: [...bundleAnalyzer],
```

Comment:
nit: falsey plugins will be ignored, so you could do it like this too


```suggestion
  const bundleAnalyzer =
    isProductionBuild || !isIntegration
      ? bundleAnalyzerPlugin({ name: 'client' })
      : null;

return createSkuViteConfig(
{
      plugins: [bundleAnalyzer],
```